simulator/infra/ports/module.cpp

```diff
@@ -41,9 +41,84 @@ void Module::enable_logging_impl( const std::unordered_set<std::string>& names)
         c->enable_logging_impl( names);
 }
 
+void Module::module_dumping( pt::ptree& pt_modules) const
```

Comment:
According to our code conventions, we use raw pointers instead of references for arguments which return data.

Reply:
I don't think we need that "pt_" prefix everywhere, the type is pretty clear here.